locations/categories.py

```diff
@@ -201,6 +203,8 @@ def get_category_tags(source) -> {}:
     for top_level_tag in top_level_tags:
         if v := tags.get(top_level_tag):
             categories[top_level_tag] = v
+    if len(categories.keys()) > 1 and categories.get("shop") == "yes":
```

Comment:
Is it worth adding a test for this change?

Reply:
Yeah, will do tomorrow